src/pint/utils.py

```diff
@@ -1686,6 +1694,103 @@ def remove_dummy_distance(c):
         return c
 
 
+def info_string(prefix_string="# ", comment=None):
+    """Returns an informative string about the current state of PINT.
+
+    Adds:
+    
+    * Creation date
```

Comment:
There's a Sphinx/docutils list construct, I think it's lines starting with -, not single-spaced, with a blank line above and below.

Reply:
I thought I was doing that.  Lines starting with "-" seem to be the same as those starting with "*".  I'm basing this off https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#bullet-lists (as much as I can interpret it).

Reply:
Ah okay it's not working, but there definitely shouldn't be blank lines between list items. (List items that extend onto more than one line would be indented to align with the first character after the *).

Reply:
https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#lists-and-quote-like-blocks this is what I followed and it seems to work